index.js

```diff
@@ -71,6 +72,11 @@ var Cordova = function(id, emitter, args, logger, config) {
           }
           var platforms = self.settings.platforms;
           var plugins = self.settings.plugins;
+          var mode = self.settings.mode;
```

Comment:
Similarly for mode, let's have a default be "emulate" in case the developer doesn't choose one.